examples/hybrid/sphere/balanced_flow_rhoe.jl

```diff
@@ -4,32 +4,27 @@ using ClimaCore.DataLayouts
 
 include("baroclinic_wave_utilities.jl")
 
-const sponge = false
-
-# Variables required for driver.jl (modify as needed)
-horizontal_mesh = cubed_sphere_mesh(; radius = R, h_elem = 4)
-npoly = 4
-z_max = FT(30e3)
-z_elem = 10
-t_end = FT(60 * 60)
-dt = FT(5)
-dt_save_to_sol = FT(50)
-dt_save_to_disk = FT(0) # 0 means don't save to disk
```

Comment:
where does the `dt_save_to_disk` go now?

Reply:
Oh, it still goes in the same place as `dt_save_to_sol`. I'll add the `dt_save_to_sol = FT(0)` back into every test case file to clarify that.

Reply:
Also, does `save_to_disk` only refers to JLD2 outputs? Is the interface compatible with the other output/visualization options we have? 
I'm honestly a bit lost in the way these files have been modified to account for pre-and-post-processing options

Reply:
`dt_save_to_disk` toggles how frequently restart files are generated. Currently, we only support JLD2 restart files, so that's the only option. All other outputs must be generated in the `postprocessing` function, which is defined in every test-case file.

Reply:
In general, a "driver setup" contains all information needed before/during a simulation run, and "postprocessing" is everything that occurs after a simulation has been run.

Reply:
@dennisYatunin as we discussed today, it would be nice if these setups and file organization could be documented. Let's add a README.md in this `example/hybrid` directory to explain where things are or need to be modified for pre-and-postprocessing. Thanks!

Reply:
Otherwise it is currently a bit hard to follow the flow of the program (what is common to all test cases, what needs to be setup for each test  case, etc...)